TunnelVisionLabs.ReferenceAssemblyAnnotator/WellKnownTypes+MemberNotNullAttributeProvidedType.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace TunnelVisionLabs.ReferenceAssemblyAnnotator
+{
+    using System;
+    using Mono.Cecil;
+    using Mono.Cecil.Rocks;
+
+    internal partial class WellKnownTypes
+    {
+        private sealed class MemberNotNullAttributeProvidedType : ProvidedAttributeType
+        {
+            public MemberNotNullAttributeProvidedType()
+                : base("System.Diagnostics.CodeAnalysis", "MemberNotNullAttribute")
+            {
+            }
+
+            protected override void ImplementAttribute(ModuleDefinition module, TypeDefinition attribute, WellKnownTypes wellKnownTypes, CustomAttributeFactory attributeFactory)
+            {
+                var constructor1 = MethodFactory.Constructor(wellKnownTypes.TypeSystem);
```

Comment:
It's the first time I play with Cecil, so you'll likely want to triple-check I'm not doing anything stupid 😄